File: examples/csharp/src/ActivityExample/ActivityExampleStack.cs
using Amazon.CDK;
using Get2Know.CDK.TwitterActivity;

namespace ActivityExample
{
    public class ActivityExampleStack : Stack
    {
        internal ActivityExampleStack(Construct scope, string id, IStackProps props = null) : base(scope, id, props)
        {
            new EventSource(this, "ActivityEventSource");
        }
    }
}

File: examples/typescript/lib/activity-example-stack.ts
import * as cdk from '@aws-cdk/core';
import * as activity from '../../../lib/index'

export class ActivityExampleStack extends cdk.Stack {
  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
    super(scope, id, props);

    const source = new activity.EventSource(this, 'ActivityEventSource')

  }
}
